pkgs/by-name/li/linux-show-player/package.nix

```diff
@@ -0,0 +1,85 @@
+{
+  lib,
+  python3,
+  fetchFromGitHub,
+  qt5,
+  wrapGAppsHook3,
+  gobject-introspection,
+  libjack2,
+  ola,
+  nix-update-script,
+}:
+let
+  version = "0.6.4";
+in
+python3.pkgs.buildPythonApplication {
+  pname = "linux-show-player";
+  inherit version;
+  pyproject = true;
+
+  # 3.13 is unsupported for now
+  disabled = with python3.pkgs; pythonOlder "3.8" || pythonAtLeast "3.13";
+
+  src = fetchFromGitHub {
+    owner = "FrancescoCeruti";
+    repo = "linux-show-player";
+    tag = "v${version}";
+    hash = "sha256-HDL3oL2N3nneXs37N/UPil8RwN1pC0sU23xVtGfOrb0=";
```

Comment:
```suggestion
    hash = "sha256-QgsgG+SeHT9bZgSkpo5AKpm/YIifn7qRfe/C3paMS5o=";
```

Reply:
Did they move the tag or something...? Bleh